recipes/inferno/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.1.8" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+
+package:
+    name: inferno
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/inferno-pytorch/inferno
+    git_tag: master
+    # TODO checkout url instead
+    # url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+
+build:
+    number: 0
+    skip: True  # [not py36 or not linux]
+    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+    host:
+        - python
+        - pip
+        - pytest-runner
+    run:
+        - python
+        - pyyaml
+        - scipy
+        - scikit-image
+        - h5py
+        - dill
+        - networkx 1.11
+        - tensorboardx
+        - pytorch >=0.4,<0.5
+        - torchvision >=0.2,<0.3
+
+test:
+    imports:
+        - inferno
+    # don't run tests for now
+    #source_files:
+    #- tests/test_inferno.py
+    #commands:
+    #- python -m unittest discover -v -s tests
+
+about:
+    license: Apache 2.0
+    license_file: LICENSE
+    license_family: APACHE
+    summary: A utility library around PyTorch
+    home: https://github.com/inferno-pytorch/inferno
+
+extra:
+    recipe-maintainers:
+        - constantinpape
+        - nasimrahaman
```

Comment:
@nasimrahaman , do you accept to be a maintainer of this recipe as well?

Reply:
Yes. :+1: